test/README.md

```diff
@@ -226,109 +228,109 @@ Tests whether `name` and `expected` are part of a raised warning.
 
 Returns an instance of all possible `ArrayBufferView`s of the provided Buffer.
 
-### hasCrypto
+#### hasCrypto
 * return [&lt;Boolean>](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Data_structures#Boolean_type)
 
 Checks for 'openssl'.
 
-### hasFipsCrypto
+#### hasFipsCrypto
 * return [&lt;Boolean>](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Data_structures#Boolean_type)
 
 Checks `hasCrypto` and `crypto` with fips.
 
-### hasIPv6
+#### hasIPv6
 * return [&lt;Boolean>](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Data_structures#Boolean_type)
 
 Checks whether `IPv6` is supported on this platform.
 
-### hasMultiLocalhost
+#### hasMultiLocalhost
```

Comment:
If we're editing these, could we move the h* properties so that they're in alphabetical order?